src/containers/Charts/Chart.test.js

```diff
@@ -1,27 +1,157 @@
-// import React from 'react';
-// import { createStore } from 'redux';
-// import { Provider } from 'react-redux';
-// import { render, cleanup } from '@testing-library/react'
-// import '@testing-library/jest-dom/extend-expect'
-// import Chart from './Chart'
-// import { rootReducer } from '../../reducers/index'
-// import { BrowserRotuer } from 'react-router-dom'
-
-// afterEach(cleanup);
-
-// const testStore = createStore(rootReducer);
-
-// const chartTestWrapper = () => {
-//   return render(
-//     <Provider store={ testStore }>
-//     <BrowserRouter>
-//     <Chart / >
-//     </BrowserRouter>
-//   </Provider>
-//   )
-// }
-
-
-describe('example', () => {
-  it('should render the correct content', () => {})
+
+import React from 'react';
+import { render, fireEvent, getAllByRole } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import rootReducer from '../../reducers/index';
+import Chart from './Chart'
+import { Bar, Line, Pie } from 'react-chartjs-2'
+jest.mock('react-chartjs-2', () => ({
+  Bar: () => null,
+  Line: () => null,
+  Pie: () => null
+}));
+
+const graphData = {
+  labels: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'],
+  datasets: [{
+    label: 'Solar Radiation',
+    backgroundColor: 'rgba(75,192,192,1)',
+    borderColor: 'whitesmoke',
+    borderWidth: 2,
+    data: [5.084908962249756,
+      6.171108722686768,
+      7.394773483276367,
+      7.778176307678223,
+      8.63671875,
+      9.294778823852539,
+      8.980157852172852,
+      8.637933731079102,
+      7.904896259307861,
+      6.774695873260498,
+      15.589939594268799,
+      14.561224937438965
+    ]
+  }]
+}
+
+
+const testStore = createStore(rootReducer);
+
+const chartTestWrapper = () => {
```

Comment:
This component was definitely a headache to try and test.   Just testing for the elements on the page without the charts should be fine

Reply:
I agree